src/app/app.routes.ts

```diff
@@ -1,18 +1,24 @@
-/* eslint-disable jsdoc/require-jsdoc */
+/**
+ * @license Apache-2.0
+ * @copyright The GHGA Authors
+ */
+
 import { inject } from '@angular/core';
 import { Routes } from '@angular/router';
 import { AuthService } from '@app/auth/services/auth.service';
 
 export const routes: Routes = [
   {
     path: '',
+    // eslint-disable-next-line jsdoc/require-jsdoc
```

Comment:
Not in the scope of this PR, but is it possible/reasonable to disable the rule for all fat arrow functions (or at least for those without code blocks)?

Reply:
I have fixed this in the latests commit. It is defined in the ESLint Config (ArrowFunctionExpression: false - used to be true)